portfolio/src/main/webapp/index.html

```diff
@@ -4,53 +4,40 @@
     <meta charset="UTF-8">
     <title>Ryan's Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script>
-
-        function addRandomFact() {
-            const facts = 
-                ['hair', 'height', 'lakers', 'hammock', 'ucsb', 'italian'];
-
-            // Pick a random fact.
-            const fact = facts[Math.floor(Math.random() * facts.length)];
-
-            // Add it to the page.
-            const showFact = document.getElementById(fact);
-            showFact.style.display = "block";
-        }
-
-    </script>
+    <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
       <h1>Ryan's Portfolio</h1>
+      <img src="images/IMG-3136.jpg" class="mediumImg"/>
       <p>Hello! My name is Ryan Simone and I am currently a Google Step Intern.</p>
       <p>I am a rising junior at the University of California Santa Barbara pursuing a Bachelor's degree in Computer Science.</p>
       <p>I'm the youngest, yet tallest, member of my family. I've lived in Southern California my entire life.</p>
       <h2>Hobbies</h2>
-      <dl>
-          <dt>I love watching sports. My favorite is soccer.</dt>
-          <dt>I enjoy skateboarding and surfing.</dt>
-          <dt>I listen to music constantly. My favorite genre is surf rock.</dt>
-      </dl>
+      <ul>
+          <li>I love watching sports. My favorite is soccer.</dt>
```

Comment:
If you're changing to li you'll need to /li everywhere
```suggestion
          <li>I love watching sports. My favorite is soccer.</li>
```